src/components/AddDestinationForm.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+
+export default function AddDestinationForm({ cities, onAddNewCity }) {
+  //   const [input, setInput] = useState("");
+  const [newDestination, setNewDestination] = useState({
+    name: "",
```

Comment:
have a look at https://rjroopal.medium.com/best-practices-for-clean-react-apps-5a891cbe48ab
point 19 regarding. the usage of quotes ' vs "